src/main.cs

```diff
@@ -119,8 +119,10 @@ public class AboutProgramm : IEplAction
 
         public bool Execute(ActionCallingContext ctx)
         {
-            MessageBox.Show($"Версия надстройки - {GetVersion()}", 
-                "Версия надстройки", MessageBoxButtons.OK, 
+            string messageLicense = $"Версия надстройки -{GetVersion()}\n" +
+               "Проект распространяется под лицензией - MIT";
+            MessageBox.Show(messageLicense,
```

Comment:
```suggestion
            MessageBox.Show(message,
```
Переименовать т.к переименовали раньше